tests/userschema.test/t09.req

```diff
@@ -0,0 +1,26 @@
+select * from test
+select * from comdb2_tablepermissions order by tablename
+
+set user dcba
```

Comment:
Can you also add test for quoted user/password as well?

Reply:
ok.